### PR TITLE
Remove credentials

### PR DESCRIPTION
**Pull request by BelehovEgor**
http://outsourcing.nat.tepkom.ru:20080/redmine/issues/677

### CODE REVIEW

**Review by TIove — APPROVED** [review]

File: Tests.HealthCheck/appsettings.json [hunk 1]
```diff
@@ -10,12 +10,6 @@
...
     "EvaluationTimeInSeconds": 180
   },
-  "SmtpCredentials": {
-    "Host": "smtp.gmail.com",
-    "Port": "587",
-    "Email": "lt.digitaloffice@gmail.com",
-    "Password": "%4fgT1_3ioR"
-  },
   "SendEmailList": [
     "fdiii@mail.ru"
```
Comment (TIove) [inline Tests.HealthCheck/appsettings.json]: Change it to corporative email please

Comment (BelehovEgor) [reply Tests.HealthCheck/appsettings.json]: This email exists?

Comment (TIove) [reply Tests.HealthCheck/appsettings.json]: That’s mine email

Comment (BelehovEgor) [reply Tests.HealthCheck/appsettings.json]: deleted

File: Tests.HealthCheck/ReportEmailSender.cs [hunk 2]
```diff
@@ -65,7 +68,7 @@ public static void SendEmails()
                 }
                 catch (Exception )
                 {
-
```
Comment (MariyaSin) [inline Tests.HealthCheck/ReportEmailSender.cs:68]: Why Exception is empty?

**Review by MariyaSin — APPROVED** [review]

File: Tests.HealthCheck/ReportEmailSender.cs (hunk 2 above)
Comment (BelehovEgor) [reply Tests.HealthCheck/ReportEmailSender.cs:68]: It is not my code and my task, I only have to remove credentials

**Review by djino97 — APPROVED** [review]

**Review by TIove — APPROVED** [review]

**Review by borovHokage — APPROVED** [review]

**Pull request merged**